### PR TITLE
bazarr: initial release (0.9.1.1)

### LINKED ISSUE #3535: [Package Request] Bazarr
Name: Bazarr
Description: Bazarr is a companion application to Sonarr and Radarr. It manages and downloads subtitles based on your requirements. You define your preferences by TV show or movies and Bazarr takes care of everything for you.

Website: https://github.com/morpheus65535/bazarr
Source code: https://github.com/morpheus65535/bazarr
License: GNU GPL v3

### PR DESCRIPTION
**Pull request by smaarn**
_Motivation:_  Creating initial bazarr spk
_Linked issues:_  Fixes #3535.

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

**Comment by smaarn** [comment]
Corresponding binaries have been published here for those interested in testing on their platforms: https://github.com/smaarn/spksrc/releases/tag/bazarr-0.9.1.1-1

(Validated on evansport architecture for what it's worth).

File: spk/bazarr/Makefile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+SPK_REV = 1
+
+SPK_ICON = src/bazarr.png
+
+BUILD_DEPENDS = cross/python38
+
+DEPENDS = cross/$(SPK_NAME)
+
+WHEELS = 
+SPK_DEPENDS = "python38:ffmpeg"
```
Comment (ymartin59) [inline spk/bazarr/Makefile:13]: Sure about it? isn't it "python38,ffmpeg" ?

Comment (hgy59) [reply spk/bazarr/Makefile:13]: all packages have `:` separator for SPK_DEPENDS, and so have synology.com packages

Comment (smaarn) [reply spk/bazarr/Makefile:13]: To be honest it seemed to work on my side and it seemed to be the same for other spks so I didn't go any further than this.

File: spk/bazarr/src/bin/bazarr-daemon [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+export LC_ALL=en_US.utf8
+export LANG=en_US.utf8
+
+if [ -f "${VAR_DIR}/profile" ]; then
+	source "${VAR_DIR}/profile"
+fi
+
+exec python3 bazarr.py --no-update --config "${VAR_DIR}/data">>"${VAR_DIR}/bazarr.log" 2>&1
```
Comment (ymartin59) [inline spk/bazarr/src/bin/bazarr-daemon:18]: I would add some spaces around ">>" for readability

**Review by ymartin59 — APPROVED** [review]

File: spk/bazarr/src/bin/bazarr-daemon (hunk 2 above)
Comment (smaarn) [reply spk/bazarr/src/bin/bazarr-daemon]: Done.

**Comment by smaarn** [comment]
Superseded by #4547

**Pull request closed without merging**